### ISSUE TITLE
Connections HAS and AS

### ISSUE THREAD

**Issue by Sanne512** [quintel/etmoses#984]
The HAS and AS are delivery stations and can be connected to all pressure levels from 100 mbar to 8 bar. The HAS are used by bulk customers (mostly industry) and the AS are used for "normal customers". At the moment I can only choose a HAS or AS at the 100 mbar level. Is it possible to also make these connectors available on the other pressure levels? This is similar to connecting customers to the MV net in electricity.

**Comment by ChaelKruip** [comment]
>  Is it possible to also make these connectors available on the other pressure levels? 

Yes, but the load calculation will not feed in the gas at those levels (yet). 

I propose to make them available at 4 and 8 bar as well. Assigning @grdw to do this.

**Comment by Sanne512** [comment]
> Yes, but the load calculation will not feed in the gas at those levels (yet).< 

This makes sense. 

> I propose to make them available at 4 and 8 bar as well.<

I agree.

**Comment by Sanne512** [comment]
What's the status?

**Comment by ChaelKruip** [comment]
> What's the status?

This feature has not been assigned to a sprint yet. Perhaps we can discuss the possibilities to do this offline @Sanne512? Also, what is the relative priority of this?

Moreover, there are some more conceptual questions that we have to answer first:
* How are we going to associate a technology/customer with a higher level of the gas network? Currently, as connection to the gas network is always associated with an end-point of the electricity topology.
  * Possible solution direction: perhaps we could add a new attribute to the topology YAMLs to specify at which pressure level this end-point feeds in and consumes.
* What do we expect the gas-calculator to do? When consumption can happen on different pressure levels simultaneously, we need to specify which pressure level operations are treated first in the calculator.

**Comment by Sanne512** [comment]
Summary of @ChaelKruip and my phone call: 

> perhaps we could add a new attribute to the topology YAMLs to specify at which pressure level this end-point feeds in and consumes

We both like this: adding customer connections (the HAS and AS) at higher pressure levels is a feature for PRO users. Thus the default remains connecting the customers to 0.125 bar pressure level. But a new attribute in the topology YAMLs could make it possible to connect the endpoint to a higher pressure level. 

> What do we expect the gas-calculator to do? When consumption can happen on different pressure levels simultaneously, we need to specify which pressure level operations are treated first in the calculator.

By using the demand side approach it makes sense to treat the lowest pressure level operations first in the calculator. 

> what is the relative priority of this

I'll let you know next week

**Comment by Sanne512** [comment]
> I'll let you know next week

For us this has medium priority. The issue is already included in the overview ticket for sprint 5. 

I'm including @MarjoleinSchwachofer and @ThomBuijs  for further reference.
